cluster/charts/crossplane/templates/secret.yaml

```diff
@@ -42,4 +42,31 @@ metadata:
   name: ess-client-certs
   namespace: {{ .Release.Namespace }}
 type: Opaque
-{{- end }}
\ No newline at end of file
+{{- end }}
+---
+# The reason this is created empty and filled by the init container is we want
+# to manage the lifecycle of the secret via Helm.
```

Comment:
Nit: I know this is copying an existing comment, but I don't really understand what it's saying. Why do we want to manage the lifecycle via Helm?

Reply:
We want to delete these certificates when we delete Crossplane. Therefore, they're created via helm.

Reply:
It might be worth updating the comment(s) to be more explicit around that.